src/create-file/index.spec.js

```diff
@@ -0,0 +1,18 @@
+process.env.RABBITHOST = '46.4.8.29';
+import callbackWorker from './index';
+
+test('if function exists', () => {
+    expect(callbackWorker).toBeDefined();
+});
+
+test('if worker provides correct outputs', async () => {
+    const job = {};
+    const outputPath = '/tmp/README.md';
+    await callbackWorker(job, [
+        'https://raw.githubusercontent.com/klips-project/rabbitmq-worker/main/README.md',
```

Comment:
shouldn't this value be one of `String|Buffer|TypedArray|DataView` ?

Reply:
it is a string?

Reply:
but then the URL would written in the text document. 
I would find it more logical to write a text like "lorem ipsum" as content to the text, otherwise people think there is a download going on